source/plugins/ruby/in_kube_nodes.rb

```diff
@@ -603,6 +603,10 @@ def getNodeTelemetryProps(item)
         end
         properties["NODES_CHUNK_SIZE"] = @NODES_CHUNK_SIZE
         properties["NODES_EMIT_STREAM_BATCH_SIZE"] = @NODES_EMIT_STREAM_BATCH_SIZE
+        nodeLabels = item["metadata"]["labels"]
+        if !nodeLabels.nil? && !nodeLabels.empty? && nodeLabels.key?("kubernetes.azure.com/fips_enabled") && nodeLabels["kubernetes.azure.com/fips_enabled"] == "true"
```

Comment:
nodeLabels.key? might not be needed, as nil == "true" will return false

Reply:
yes, its not needed but checking there is no harm as well.